src/App.js

```diff
@@ -1,22 +1,35 @@
 import React from 'react';
 import './App.css';
-
 import todos from './api/todos';
 import users from './api/users';
+import TodoList from './api/components/TodoList';
+
+const preparedTodos = todos.map((item) => {
+  const personTodoList = { ...item };
+
+  personTodoList.user = users.find(person => (person.id === item.userId));
+
+  return personTodoList;
+});
```

Comment:
```suggestion
const preparedTodos = todos.map((item) => (
  {
    ...item,
    user: users.find(person => (person.id === item.userId)),
  }
));
```
Можно использовать короткую запись

Reply:
fixed